Assets/MixedRealityToolkit/_Core/Utilities/Scripts/ClipPlane.cs

```diff
@@ -32,18 +32,16 @@ private void OnDisable()
             ToggleClippingPlane(false);
         }
 
-#if UNITY_EDITOR
         private void Update()
         {
-            if (Application.isPlaying)
+            if (Application.isPlaying || Application.isEditor)
```

Comment:
should this be && instead of || ? The original code got executed when it was in the editor, this looks like it will occur when the app is running on device as well.

Reply:
|| is correct, but I needed to add a !

Should be `if (Application.isPlaying || !Application.isEditor)`